open-api.yaml

```diff
@@ -892,59 +860,16 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/dumps'
+                $ref: '#/components/responses/202'
               examples:
-                Created:
+                '202':
                   value:
-                    uid: 20210719-114144097
-                    status: in_progress
-                    startedAt: '2021-07-19T14:31:16.920473Z'
+                    uid: 0
```

Comment:
This should have a `type`, right?

Reply:
Good point Maryam. I'll change it to the appropriate place! Thank you very much.